compiler/dyno/lib/uast/chpl-syntax-printer.cpp

```diff
@@ -712,47 +880,57 @@ struct ChplSyntaxVisitor {
       printChapelSyntax(ss_, node->condition());
       ss_ << " ";
     }
-    printBlockWithStyle(node->blockStyle(), node->stmts(), "do ");
+    printBlockWithStyle(node->blockStyle(), node->stmts(), "do ", ";", true);
   }
 
   void visit(const Manage* node) {
     ss_ << "manage ";
     interpose(node->managers(), ", ");
     ss_ << " ";
-    printBlockWithStyle(node->blockStyle(), node->stmts(), "do ");
+    printBlockWithStyle(node->blockStyle(), node->stmts(), "do ", ";", true);
   }
 
   void visit(const Module* node) {
     if (node->visibility() != Decl::Visibility::DEFAULT_VISIBILITY) {
       ss_ << kindToString(node->visibility()) << " ";
     }
-    if (node->kind() != Module::Kind::DEFAULT_MODULE_KIND ) {
+    if (node->kind() != Module::Kind::DEFAULT_MODULE_KIND && node->kind() != Module::Kind::IMPLICIT) {
       ss_ << kindToString(node->kind()) << " ";
     }
     ss_ << "module ";
     ss_ << node->name() << " ";
-    interpose(node->stmts(), "\n", "{", "}");
+    interpose(node->stmts(), "\n", "{\n", "\n}\n",";", true);
   }
 
   void visit(const MultiDecl* node) {
-    ss_ << "var ";
+    bool isConfig = false;
+    std::string kind;
+    for (auto decl : node->decls()) {
+      if (!decl->isComment()) {
+        if (decl->isVariable() ) {
+          if (decl->toVariable()->isConfig())
+            isConfig = true;
+          kind = kindToString((IntentList)decl->toVariable()->kind());
+        }
+        else if (decl->isTupleDecl()) {
+          // TODO: can tuple decls be configs?
```

Comment:
I think they definitely can be and we can parse that (IIRC?), it's just we don't currently support setting them and so don't support them as a result. They require a recursive sort of logic we just haven't gotten around to doing.